### PR TITLE
Display duration from datastore.json

### PR DESCRIPTION
**Pull request by helfi92**
Screen Shot 2020-08-07 at 12 01 53 PM

### CODE REVIEW

**Review by djmitche — COMMENTED** [review]
Looks good!  Do you want to update the README with how this should be used?

If `datastore.json` is updated, does this reload automatically?  Maybe this is enough and we don't need to do the whole `yarn analyze` thing at all!

**Comment by helfi92** [comment]
> Looks good! Do you want to update the README with how this should be used?

Done.

> If datastore.json is updated, does this reload automatically? Maybe this is enough and we don't need to do the whole yarn analyze thing at all!

Yep, it will reload automatically.

**Review by djmitche — APPROVED** [review]
Wow, I think that's fantastic!

File: README.md [hunk 1]
```diff
@@ -248,3 +248,16 @@ the available simulations are defined in `.js` files in `sims`.
...
+A web interface can be used to visualize the output of a simulation.
+
+### Requirements
+
+The UI presently expects the output of the simulation to be stored in a file named `datastore.json` in the root
+repository.
+
+### Development
+
+Run `yarn start`.
```
Comment (djmitche) [inline README.md]: I think this is now for more than development!

To visualize the output of a simulation:

in one terminal:

```
yarn sim $SIMULATION -o datastore.json
```

Then, in a new terminal, run
```
yarn start
```
and navigate to ...?.  The page will automatically reload if `datastore.json` is modified, such as if you run a new simulation.

**Pull request merged**